pallets/funding/src/tests/3_auction.rs

```diff
@@ -886,6 +886,283 @@ mod bid_extrinsic {
+
+			inst.execute(|| {
+				assert_ok!(<pallet_proxy_bonding::Pallet<TestRuntime>>::transfer_fees_to_recipient(
+					RuntimeOrigin::signed(BIDDER_1),
+					project_id,
+					HoldReason::Participation.into(),
+					USDT_ID
+				));
+				assert_noop!(
+					<pallet_proxy_bonding::Pallet<TestRuntime>>::transfer_bonds_back_to_treasury(
+						RuntimeOrigin::signed(BIDDER_1),
+						project_id,
+						HoldReason::Participation.into()
+					),
+					pallet_proxy_bonding::Error::<TestRuntime>::TooEarlyToUnlock
+				);
+			});
+			let now = inst.current_block();
+			inst.jump_to_block(otm_duration + now);
+			inst.execute(|| {
+				assert_ok!(<pallet_proxy_bonding::Pallet<TestRuntime>>::transfer_bonds_back_to_treasury(
+					RuntimeOrigin::signed(BIDDER_1),
+					project_id,
+					HoldReason::Participation.into()
+				));
+			});
+
+			let post_settlement_treasury_free_plmc = inst.get_free_plmc_balance_for(otm_treasury_account);
+			let post_settlement_otm_escrow_held_plmc = inst.get_free_plmc_balance_for(otm_escrow_account);
+			let post_settlement_otm_escrow_usdt = inst.get_free_funding_asset_balance_for(USDT_ID, otm_escrow_account);
+			let post_settlement_otm_fee_recipient_usdt =
+				inst.get_free_funding_asset_balance_for(USDT_ID, otm_fee_recipient_account);
+			let post_settlement_buyer_usdt = inst.get_free_funding_asset_balance_for(USDT_ID, BIDDER_1);
+			let issuer_funding_account = inst.get_free_funding_asset_balance_for(USDT_ID, issuer);
+
+			assert_eq!(post_settlement_treasury_free_plmc, post_participation_treasury_free_plmc + expected_plmc_bond);
+			assert_eq!(post_settlement_otm_escrow_held_plmc, inst.get_ed());
+			assert_eq!(post_settlement_otm_escrow_usdt, Zero::zero());
+			assert_eq!(post_settlement_otm_fee_recipient_usdt, OTM_USDT_FEE);
+			assert_eq!(post_settlement_buyer_usdt, usdt_ed);
+			assert_eq!(issuer_funding_account, USDT_PARTICIPATION);
+		}
+
+		#[test]
+		fn one_token_mode_bid_funding_failed() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let issuer = ISSUER_1;
+
+			let mut project_metadata = default_project_metadata(issuer);
+			project_metadata.mainnet_token_max_supply = 50_000 * CT_UNIT;
+			project_metadata.total_allocation_size = 20_000 * CT_UNIT;
+			project_metadata.minimum_price = <TestRuntime as Config>::PriceProvider::calculate_decimals_aware_price(
+				PriceOf::<TestRuntime>::from_float(1.0),
+				USD_DECIMALS,
+				CT_DECIMALS,
+			)
+			.unwrap();
+
+			let evaluations =
+				inst.generate_successful_evaluations(project_metadata.clone(), default_evaluators(), default_weights());
+
+			let project_id = inst.create_auctioning_project(project_metadata.clone(), issuer, None, evaluations);
+			let otm_multiplier: MultiplierOf<TestRuntime> =
+				ParticipationMode::OTM.multiplier().try_into().ok().unwrap();
+
+			const USDT_ID: u32 = AcceptedFundingAsset::USDT.id();
+			const USDT_PARTICIPATION: u128 = 5000 * USDT_UNIT;
+
+			const OTM_USDT_FEE: u128 = 50 * USDT_UNIT;
```

Comment:
We manually have to change this value if we change the FeePercentage, right? Is it possible to do something like `OTM_USDT_FEE = USDT_PARTICIPATION * FeePercentage`?